node/cli/package.json

```diff
@@ -21,6 +21,6 @@
     "typescript": "^4.5.5",
     "@ledgerhq/hw-transport-node-hid": "6.20.0",
     "@ledgerhq/hw-transport-node-speculos": "6.20.0",
-    "hw-app-obsidian-common": "git+https://github.com/obsidiansystems/hw-app-obsidian-common"
+    "hw-app-obsidian-common": "git+https://github.com/obsidiansystems/hw-app-obsidian-common#wip_new_protocol"
```

Comment:
Whoever wants to merge this, just pointing out that the downstream should be merged first, and this PR updated.